php-classes/Slate/Courses/SectionsRequestHandler.php

```diff
@@ -110,11 +111,52 @@ public static function handleRecordRequest(ActiveRecord $Section, $action = fals
                 ]));
             case 'students':
                 return static::handleStudentsRequest($Section);
+            case false:
+                return static::handleBlogpostRequest($Section, $action);
             default:
                 return parent::handleRecordRequest($Section, $action);
         }
     }
 
+    public static function handleBlogpostRequest(Section $Section, $action)
```

Comment:
```suggestion
    public static function handleBlogpostRequest(Section $Section, $action)
```

For one, we'd want to call this `handleBlogPostRequest` to match how BlogPost is usually cased in Slate, but also this is `handleSectionRequest` -- the primary response payload and thing being requested is still a section, we just may or may not include related blog post data